src/Microsoft.DotNet.Build.Tasks/PackageFiles/buildagainstpackages.targets

```diff
@@ -3,9 +3,11 @@
   <UsingTask TaskName="AddDependenciesToProjectJson" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
   <UsingTask TaskName="GetPackageNumberFromPackageDrop" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
   <UsingTask TaskName="DumpItem" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
+  <UsingTask TaskName="FilterRuntimesFromSupports" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
 
   <ItemGroup>
     <_PackagesDrops Include="$(PackagesDrops)" />
+    <SupportsDefinitionFile Condition="'$(SupportsDefinitionFile)' == '' AND Exists('$(SrcDir)Microsoft.DotNet.BuildTools.TestSuite/runtime.json')">$(SrcDir)Microsoft.DotNet.BuildTools.TestSuite/runtime.json</SupportsDefinitionFile>
```

Comment:
SrcDir was in corefx, but this is the wrong default. It should be the runtime.json in the packagesdir.